schnorr_fun/src/musig.rs

```diff
@@ -124,78 +125,112 @@ pub struct KeyList {
     agg_key: Point<Normal>,
     /// The tweak on the aggregate key
     tweak: Scalar<Public, Zero>,
-    /// Whether this aggregate key needs negation.
-    accum_needs_negation: bool,
 }
 
-impl KeyList {
-    /// The `XOnly` aggregated key for the keylist.
-    pub fn agg_public_key(&self) -> XOnly {
-        self.agg_key.to_xonly()
+impl AggKey {
+    /// The aggregate key prior to converting to a BIP340 `XOnly` key.
+    pub fn agg_key(&self) -> Point {
+        self.agg_key
     }
-    /// The aggregated key for the keylist as a `Point`.
-    pub fn agg_verification_key(&self) -> Point<EvenY> {
-        let (k, _) = self.agg_key.into_point_with_even_y();
-        k
-    }
-    /// An iterator over the **public keys** of each party in the keylist.
+
+    /// An iterator over the **public keys** of each party in the aggregate key.
     pub fn keys(&self) -> impl Iterator<Item = XOnly> + '_ {
         self.parties.iter().map(|xonly| *xonly)
     }
 
-    /// Tweak the aggregate MuSig public key with a scalar so that the resulting key is equal to the
-    /// existing key plus `tweak * G`. The tweak mutates the public key while still allowing
-    /// the original set of signers to sign under the new key.
+    /// Add a scalar `tweak` to aggregate MuSig public key.
```

Comment:
Both tweaks functions take scalar tweaks as inputs so perhaps this isn't the best description. But I also do not like `ordinary` tweaks..